kotsadm/pkg/apiserver/server.go

```diff
@@ -93,7 +93,7 @@ func Start() {
 	// Implemented handlers
 	r.Path("/api/v1/license/platform").Methods("OPTIONS", "POST").HandlerFunc(handlers.ExchangePlatformLicense)
 	r.Path("/api/v1/app/{appSlug}/sequence/{sequence}/preflight/ignore-rbac").Methods("OPTIONS", "POST").HandlerFunc(handlers.IgnorePreflightRBACErrors)
-	r.Path("/api/v1/app/{appSlug}/preflight/run").Methods("OPTIONS", "POST").HandlerFunc(handlers.StartPreflightChecks)
+	r.Path("/api/v1/app/{appSlug}/sequence/{sequence}/preflight/run").Methods("OPTIONS", "POST").HandlerFunc(handlers.StartPreflightChecks)
```

Comment:
@divolgin calls to this route were removed in this pr, so it should be safe to modify https://github.com/replicatedhq/kots/pull/713/files

The commit which added this route: https://github.com/replicatedhq/kots/commit/c9e1f635